src/devices/Ad7193/Ad7193.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <Configurations>Windows-Debug;Linux-Debug;Linux-Release;Windows-Release;Debug;Release</Configurations>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--Cases used when running in VS, with the solution config either Windows-Debug or Linux-Debug (or -Release) -->
```

Comment:
Please also remove all these next lines and just keep the ProjectReference to System.Device.Gpio which is passing in the RuntimeIdentifier as Linux. You can take a look at our other projects to see what do we usually do.